Add action acf_to_rest_api_init to plugin init()

digitaldesigndj opened the pull request:
https://github.com/airesvsg/acf-to-rest-api/issues/193

This is all I need. Then I can:

```
add_filter( 'acf_to_rest_api_init', function () {
	require_once( get_template_directory() . '/acf/acf.php' );
});
```

And the plugin is fixed! 🍾

airesvsg commented:
Hello @digitaldesigndj!

Thanks for your PR!

**Why not use priority?**

```PHP
add_action( 'rest_api_init', 'your_function', 9 );
```

`add_action` parameters reference:
https://developer.wordpress.org/reference/functions/add_action/#parameters

Line in my plugin with the action:
https://github.com/airesvsg/acf-to-rest-api/blob/master/class-acf-to-rest-api.php#L72

Thanks

digitaldesigndj commented:
I think my PR is wrong. `require_once( get_template_directory() . '/acf/acf.php' );` only works when called before `includes()` does is_plugin_active()`.

You example above does not include the `acf` class early enough. That's why I think we need an action. I'm not sure where to put it or name it though

digitaldesigndj commented:
I changed it up so it's actually working now 🤐 Also changed the hook name to: `acf_to_rest_api_custom_acf_loader`

so I load the `acf` class like this now:
```
add_filter( 'acf_to_rest_api_custom_acf_loader', function () {
	require_once( get_template_directory() . '/acf/acf.php' );
});
```
